salt/orchestrate/aws/map_templates/instance_map.yml

```diff
@@ -10,10 +12,12 @@
       tag:
         environment: {{ environment_name }}
         role: {{ service_name }}
+        created_at: {{ salt.status.time(format=ISO8601) }}
```

Comment:
AWS has 'Launch Time' for EC2 instances, so wouldn't this be redundant or is the idea to have the same value across all objects in AWS?